Trading/Exchange/bybit/bybit_exchange.py

```diff
@@ -208,6 +214,12 @@ def fix_order(self, raw, **kwargs):
         if tigger_above := order_info.get(trading_enums.ExchangeConstantsOrderColumns.TRIGGER_ABOVE.value):
             fixed[trading_enums.ExchangeConstantsOrderColumns.TRIGGER_ABOVE.value] = \
                 tigger_above == self.BYBIT_TRIGGER_ABOVE_VALUE
+        status = fixed[trading_enums.ExchangeConstantsOrderColumns.STATUS.value]
+        try:
+            status = trading_enums.OrderStatus(status)
+        except ValueError:
+            if status == self.PARTIALLY_FILLED_CANCELED:
+                status = trading_enums.OrderStatus.PARTIALLY_FILLED.value
```

Comment:
The current OctoBot behavior for PARTIALLY_FILLED is to consider the order filled. Does this behavior seem correct to you for this status?